reports/satwikkansal.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: report
+permalink: /GSoC/2017/StatusReport/satwikkansal
+student: Satwik Kansal
+organisation: coala
+organisation_link: https://coala.io
+project: Enhance coala-quickstart
+project_link: https://summerofcode.withgoogle.com/projects/#4853947354316800
+tarball: https://rawgit.com/satwikkansal/GSoC-Report/master/patches.tar
+mentors: >
+ [Adhityaa](https://github.com/adtac) & [Zatreanu Adrian-Gabriel](https://github.com/Adrianzatreanu)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC-2017/milestones/6
+ - Phase 1 : https://gitlab.com/coala/GSoC-2017/milestones/20
+ - Phase 2 : https://gitlab.com/coala/GSoC-2017/milestones/21
+ - Phase 3 : https://gitlab.com/coala/GSoC-2017/milestones/22
+bio: >
+ I'm a final year Software Engineering undergrad from Delhi Technological University, New Delhi, India. I was a Google Summer of Code student for coala over the summer of 2017 and primarily worked on the tool [coala-quickstart](https://github.com/coala/coala-quickstart). My project involved developing an information extraction & utilization framework for project files and improving the existing CLI of quickstart.
+
+social:
+  - GitHub:
+    - username: satwikkansal
+    - link: https://github.com/satwikkansal
+  - GitLab:
+    - username: satwikkansal
+    - link: https://gitlab.com/satwikkansal
+  - Gitter:
+    - username: satwikkansal
+    - link: https://gitter.im/satwikkansal
+email: satwikkansal@gmail.com
+blog: https://satwikkansal.xyz/archives
+activity:
+- 0
+ - repo: cEPs
+ - link: https://github.com/coala/cEPs/commit/eb4f194fe8b4cdede65061c08f880c2880936332
+ - details: cEP 9꞉ Extract and use information
+- 1
+ - repo: coala-quickstart
+ - link: https://github.com/coala/coala-quickstart/commit/de552a2df32aab56821a2fc0577beada15c7c0bf
+ - details: CI꞉ Add Travis
+- 2
+ - repo: coala-quickstart
+ - link: https://github.com/coala/coala-quickstart/commit/a33c3b1239543c6acfa4cfac849fb5448be86fdf
+ - details: info_extraction꞉ Add Info class
+- 3
+ - repo: coala-quickstart
+ - link: https://github.com/coala/coala-quickstart/commit/7ae36300e5380bbae11918cc873ef0d4c4beff1f
+ - details: info_extraction꞉ Add InfoExtractor class
+- 4
+ - repo: coala-quickstart
+ - link: https://github.com/coala/coala-quickstart/commit/f9bd418cd35827a6b8fc7dad7c66f757ca377f78
+ - details: Project.py꞉ Remove OS specific doctest
+- 5
+ - repo: coala-quickstart
+ - link: https://github.com/coala/coala-quickstart/commit/e34a334b1084d95606ac4d9e66b145c793843915
+ - details: tests/FileGlobs.py꞉ Normalize case for path
+- 6
+ - repo: coala-quickstart
+ - link: https://github.com/coala/coala-quickstart/commit/d25f3caf9ad9cffafb8c0b5a8a33ed6c1f4fe874
+ - details: CI꞉ Add appveyor
+- 7
```

Comment:
Needs a colon here, I'm guessing I missed to add it in my script :sweat_smile:

Reply:
Ah, fixing it!

Reply:
Added colons but still not rendering :(

Reply:
@satwikkansal actually it uses  ```CI꞉``` as a key itself, it'll render inside quotes "CI: Add appveyor"

Reply:
It actually renders for meetmangukiya's report https://github.com/coala/projects/blame/master/reports/meetmangukiya.md#L43

Reply:
@hemangsk wrapped the description inside quotes, but still not rendering :sweat_smile:

Reply:
Oh yeah, I just checked in yamllint actually the nested keys are needed to be indented by2 spaces
![screen shot 2017-09-02 at 4 28 19 pm](https://user-images.githubusercontent.com/13018570/29994996-cf10e8c2-8ffb-11e7-96e2-07524e536066.png)

Reply:
I think I see what the problem is

Reply:
@satwikkansal all the `- <number>` should be indented more than `activity` key, add at least one space

Reply:
and yes, colon was not the problem, because the colon I used in the script was not a colon, but a different unicode character that looked like a colon

Reply:
Yep, that worked like a charm, thanks guys!